packages/code.rb

```diff
@@ -3,17 +3,19 @@
 class Code < Package
   description 'Visual Studio Code is a source code editor developed by Microsoft for Windows, Linux and macOS.'
   homepage 'https://code.visualstudio.com/'
-  version '1.25.1'
   case ARCH
   when 'x86_64'
+    version '1.25.1'
     source_url 'https://vscode-update.azurewebsites.net/1.25.1/linux-x64/stable'
     source_sha256 '5856bbebf38aa05d584da4722869bbe507cf123f69f7ffab5f1532d73dbf3438'
   when 'i686'
+    version '1.25.1'
     source_url 'https://vscode-update.azurewebsites.net/1.25.1/linux-ia32/stable'
     source_sha256 'af6adc2e2500e50bfebe7ee7b97d661b6e774a590136bf5f89334132a5b292e2'
   else
-    source_url 'https://raw.githubusercontent.com/Microsoft/vscode/1.25.1/README.md'
-    source_sha256 'c1c5e6ec903730a4e116c1cfd83bb29acd227195d97f2ec8944452808232c310'
+    version '1.28.0'
+    source_url 'https://github.com/fsx950223/vscode-arm/releases/download/1.28.0/code.tar.gz'
+    source_sha256 '98260cb89024e2bbd61edb5ed620f29daefc1bcbaa041cca249edc446baebe7c'
```

Comment:
We should not be removing the version in this package.  Have you tried to see if code will install via dpkg on any other architecture?  I'm not a big fan of using Debian packages to create binaries if we can successfully compile from source or use the official pre-built binaries from Azure.  Can you please describe the steps you performed to create your pre-built binary for arm?

Reply:
He's not using `dpkg`. He's using a tarball.

Reply:
I'm willing to bet money that he built this Chromebrew package by installing via dpkg.

Reply:
Dude, you don't understand what I'm asking here.  He created that tarball somehow.  I bet he installed using dpkg and then added the files to build his Chromebrew package.

Reply:
So? How is that bad? The only things Debian screws up are in the `control.tar.gz` file. That's not in there. He could've very well built it again, also.

Reply:
Anyway, @fsx950223, please build version 1.25.1 for arm, see my comments above, and this is good.

Reply:
@JL2210 I build the version 1.21.1

Reply:
Anyone could have a test.There is something wrong with my chromebook

Reply:
@uberhacker The package isn't depend on dpkg. I get the binary files from deb and compress it again.

Reply:
I have tried, but @fsx950223 when run command "code" I get this error `/usr/local/bin/code: line 35: /usr/local/share/code/bin/../code: Permission denied`